scio-core/src/main/scala/com/spotify/scio/hash/ApproxFilter.scala

```diff
@@ -276,14 +276,9 @@ sealed trait ApproxFilterCompanion {
     } else {
       val settings = partitionSettings(expectedInsertions, fpp, 100 * 1024 * 1024)
       logger.info(
-        "Partition settings for approximate filter side input of {} keys: " +
-          "partitions={}, expectedInsertions={}, sizeBytes={}",
-        Seq(
-          expectedInsertions,
-          settings.partitions,
-          settings.expectedInsertions,
-          settings.sizeBytes
-        )
+        s"Partition settings for approximate filter side input of $expectedInsertions keys: " +
+          "partitions=$settings.partitions, expectedInsertions=$settings.expectedInsertions," +
```

Comment:
This will need to be smth like:
```suggestion
          s"partitions=${settings.partitions}, expectedInsertions=${settings.expectedInsertions}," +
```

Although I think it's better if you do a multiline string here

Reply:
It's still borked on the next line. I'll fix it really quick.